src/main/java/uk/gov/pay/adminusers/resources/UserResource.java

```diff
@@ -79,6 +80,30 @@ public Response getUser(@PathParam("externalId") String externalId) {
                 .orElseGet(() -> Response.status(NOT_FOUND).build());
     }
 
+    @Path(USER_EMAILS_RESOURCE)
+    @POST
```

Comment:
Since the input to this is so simple, I wonder if it might be more straightforward to make this a `GET` and have path parameter containing a comma-separated list of external IDs. This would remove the need to check if there is a body, check that the body has a an appropriately-named property etc. It would also make using the resource a bit more ergonomic (no need to construct a body).